papermill/exceptions.py

```diff
@@ -13,6 +13,10 @@ class FileExistsError(AwsError):
     """Raised when a File already exists on S3."""
 
 
+class FileNotFoundError(AwsError):
```

Comment:
I think we shouldn't inherit from `AwsError` here as the exception is only used for local files.

Reply:
@betatim ok. Can I throw the normal Exception with the message that directory doesn't exists?

Reply:
@harsham4026 There's already a built-in FileNotFoundError in Python. Let's use that here :)